src/modules/Login/Login.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+    Button,
+    TextField,
+    Typography,
+} from '@material-ui/core'
+import firebase from 'firebase/app'
+import { useFormik } from 'formik'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import * as Yup from 'yup'
+
+import {
+    LoginContent,
+    LoginForm,
+    LoginRoot,
+} from './Login.styles'
+import type { LoginFormType } from './Login.types'
+
+const ValidationSchema = Yup.object().shape({
+    email: Yup.string()
+        .email('Must be a valid email')
+        .required('Required'),
+    password: Yup.string()
```

Comment:
You dont need to validate password in login since it can only be right or wrong, nothing else

Reply:
Be sure to remove error and helperText in password TextField since nothing will be passed in